CI/Style-To-Repo/build-and-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Style to Repo
+#
+# Copyright (c) 1998-2009 ILIAS open source, Extended GPL, see docs/LICENSE
+#
+# Author Daniel Weise <daniel.weise@concepts-and-training.de>
+#
+# Build and deploy style specific files.
+
+NOW=$(date +'%d.%m.%Y %I:%M:%S')
+echo "[${NOW}] Building style folder."
+
+source "./CI/Style-To-Repo/build.sh"
+
+source "./CI/Style-To-Repo/deploy.sh"
```

Comment:
Please use functions for the contents of these files as well and then call functions here. Remember that function names are a super cheap way of documenting code...